tests/pkgmgrs/YRoot

```diff
@@ -29,7 +29,7 @@ CustomInstaller(
 
 AptGroup(
     'ruby',
-    packages=['ruby2.3', 'ruby2.3-dev'],
+    packages=['ruby', 'ruby-dev'],
```

Comment:
This is to fix failures in Travis - apparently there is a new python docker image that doesn't have ruby2.3 in the apt repo, it has ruby instead (2.5)